### DISCUSSION TITLE
Highlight group for current scope

### DISCUSSION THREAD

**Discussion by josetaas** [LazyVim/LazyVim#2222] [Q&A] [answered]
What highlight group does the `return {` below use? Same question for the indent guide below (white vertical line).

Thanks!

<img width="1343" alt="Screenshot 2023-12-20 at 12 25 31 AM" src="https://github.com/LazyVim/LazyVim/assets/7754929/f387ddd6-dd34-41bc-9ec0-e19de987afed">

**Comment by dpetka2001** [comment, marked as answer]
If you place your curson on `{` and you press `<leader>ui` it will show you which highlight group is linked to.
The indent guide is coming from `mini.indentscope` and you can see the highlight groups in the official [docs](https://github.com/echasnovski/mini.indentscope/blob/5a8369475cd7cd6f207a4d288406d03b0fc48bdb/doc/mini-indentscope.txt#L55-L63).
